interface/forms/gad7/new.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+/**
+ * gad-7 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("gad7.inc"); //common strings, other includes etc
+use OpenEMR\Common\Csrf\CsrfUtils;    // security module
+formHeader("Form: GAD-7");
+$returnurl = 'encounter_top.php';   // update top of encounter window when done
+?>
+<html><head>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+</head>
```

Comment:
although some of the old forms have the formHeader call most new forms are using the core header, so something like
the below will bring in what you need for styling too
```
require_once("gad7.inc"); //common strings, other includes etc

use OpenEMR\Common\Csrf\CsrfUtils;    // security module
use OpenEMR\Core\Header;

$form_title = "GAD-7"
?>
<html>
<head>
    <title><?php echo $form_title; ?></title>
   <?php Header::setupHeader(); ?>
</head>
```